### PR TITLE
Temporarily use mirrors of initramfs/isc-dhcp/python-click/radvd repoes

### PR DESCRIPTION
**Pull request by qiluo-msft**
The official repo is down. So I replaced it with a mirror temporarily.

### CODE REVIEW

**Comment by dzhangalibaba** [comment]
@qiluo-msft isc-dhcp/Makefile. Has similar issue. Repo is down

**Comment by jleveque** [comment]
Why is this affecting 201911 but not master?

File: src/initramfs-tools/Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ INITRAMFS_TOOLS_REVISION = 18fc98e1b63b012f9bcf06ae3f5477872a5880c0
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# Obtaining the initramfs-tools
 	rm -rf ./initramfs-tools
-	git clone --branch v0.130 https://salsa.debian.org/kernel-team/initramfs-tools.git ./initramfs-tools
+	git clone --branch v0.130 https://github.com/qiluo-msft/initramfs-tools.git ./initramfs-tools
```
Comment (tahmed-dev) [inline src/initramfs-tools/Makefile:13]: Thanks! This is affecting internal/master build as well.

**Comment by abdosi** [comment]
Looks like issue is resolved now. Able to do `git clone --branch v0.130 https://salsa.debian.org/kernel-team/initramfs-tools.git`

**Pull request closed without merging**